_posts/developers/pod-server/2019-01-01-00_overview.md

```diff
@@ -17,11 +17,18 @@ You can either [use a publicly available server](/users/get-a-pod)
 or follow the steps on this page to set up your own.
 
 The following **open-source servers** are available:
+
 * [Community Solid Server](https://github.com/CommunitySolidServer/CommunitySolidServer/) for Node.js
 * [Node Solid Server](https://github.com/solid/node-solid-server) for Node.js
 * [solid-nextcloud](https://github.com/pdsinterop/solid-nextcloud) as a plugin for Nextcloud
 * [PHP Solid Server](https://github.com/pdsinterop/php-solid-server) for PHP
 
+And the following **enterprise servers** are available:
```

Comment:
"enterprise" should be "commercial" I think

Reply:
done.